integration_test/package.json

```diff
@@ -6,9 +6,9 @@
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "start": "react-native start",
-    "stop-test:android": "adb shell am force-stop com.bitmovin.player.reactnative.integrationtest",
-    "stop-test:ios": "xcrun simctl list devices available -e -j | jq --raw-output '.devices.[].[] | select( .state == \"Booted\" and (.deviceTypeIdentifier | contains(\"iPhone\"))).udid' | xargs -I UDID xcrun simctl terminate UDID com.bitmovin.player.reactnative.IntegrationTest 2> /dev/null; exit 0",
-    "start-test:android": "yarn cavy run-android",
+    "stop-test:android": "adb shell am force-stop com.bitmovin.player.reactnative.integrationtest; true",
```

Comment:
What does the "true" do?

Reply:
It returns with success exit code even if the previous command failed. E.g. if no emulator was running.